### PR TITLE
Feature/logging

### PR DESCRIPTION
**Pull request by tince250**
Implemented logging for Madagascar API.

Two loggers:
- security - for error messages and potential malicious behaviour
- application - for documenting all requests to the app

Logging to both console and log file.

### CODE REVIEW

File: zanzibar/app.py [hunk 1]
```diff
@@ -15,88 +21,103 @@ def get_books():
...
         return jsonify({}), 200
     except:
+        security_logger.warning(f"Bad request from IP: {request.remote_addr}")
         return jsonify({'Bad request!'}), 400
 
 @app.route('/namespace', methods=['GET'])
 @require_api_key
 def get_config():
-    namespace = request.args.get('namespace')
+    namespace = sanitize_input(request.args.get('namespace'))
```
Comment (nadazaric) [inline zanzibar/app.py:49]: Consider adding encryption before logging.

**Review by nadazaric — COMMENTED** [review]
Consider adding encryption to sensitive data before logging. Everything else looks fine :)

File: zanzibar/app.py (hunk 1 above)
Comment (tince250) [reply zanzibar/app.py:49]: Sure, added! Check it out now :)

Comment (nadazaric) [reply zanzibar/app.py:49]: Nice! I will merge now.

**Pull request merged**